smart-contracts/assembly/contracts/FT/WMAS.ts

```diff
@@ -33,9 +52,15 @@ export function withdraw(bs: StaticArray<u8>): void {
   const recipient = new Address(
     args.nextString().expect('recipient is missing'),
   );
-
-  assert(amount > 0, 'Payment must be more than 0 WMAS');
-
   burn(u256ToBytes(u256.fromU64(amount)));
   transferCoins(recipient, amount);
 }
+
+function computeStorageCost(receiver: Address): u64 {
+  if (Storage.hasOf(Context.callee(), balanceKey(receiver))) {
+    return 0;
+  }
+  return (
+    BASE_COST + VAL_COST + (7 + receiver.toString().length) * STORAGE_BYTE_COST
```

Comment:
sorry but i think its hard to understand, what is this "7" ?

Reply:
The fact that calculation is done elsewhere is also impacting readibility. I suggest to do something like:
```typescript
const baseLength = NODE_STORAGE_PREFIX_BYTES;
const keyLength = BALANCE_KEY_PREFIX.lenght() + receiver.toString().length; 
const valueLength = 4 * sizeof<u64>(); // size of a u256

return (baseLength + keyLength + valueLength) * STORAGE_BYTE_COST
```
The fact that we are taking 7 bytes just as a balance prefix is... unoptimized.